dev/io.openliberty.checkpoint_fat/bnd.bnd

```diff
@@ -38,7 +39,9 @@ tested.features:\
 	microProfile-5.0, \
 	webProfile-8.0, \
 	webProfile-9.1, \
-	ejb-2.3
+	ejb-2.3, \
+	appSecurity-3.0
```

Comment:
Is this needed?  I thought we only needed these placed for programmatic enabling features.  But you have this feature in the test `server.xml`

Reply:
removed it.